### ISSUE TITLE
"'404.html' doesn't exist" error when using ReqHandler

### ISSUE THREAD

**Issue by sagnitude** [rapidoid/rapidoid#62]
I wrote a simple application with rapidoid-web version 5.1.0: 

```
public class RapidoidRunner {

    public static void main(String[] args) {

        On.address("0.0.0.0").port(5432);

        Rapidoid.run(args);

        On.get("/showUri").json(Req::uri);
    }
}
```

But I failed to open the registered url "http://localhost:5432/test" in browser and I got the following exceptions:

```
Caused by: java.lang.IllegalArgumentException: The file '404.html' doesn't exist! Path: [, templates, default\templates]
	at org.rapidoid.u.U.illegal(U.java:386)
	at org.rapidoid.u.U.must(U.java:419)
	at org.rapidoid.io.Res.mustExist(Res.java:261)
	at org.rapidoid.render.Templates$1.map(Templates.java:39)
	at org.rapidoid.render.Templates$1.map(Templates.java:36)
	at org.rapidoid.commons.AutoExpandingMap.get(AutoExpandingMap.java:56)
	... 13 more
```

I was running it in Intellij Idea. I also tried to package everything into a single jar including the file "404.html" in "default/templates" directory. But the error remains.

Detailed error log:

```
"C:\Program Files\Java\jdk1.8.0_20\bin\java" -agentlib:jdwp=transport=dt_socket,address=127.0.0.1:25401,suspend=y,server=n -Dfile.encoding=UTF-8 -classpath "C:\Program Files\Java\jdk1.8.0_20\jre\lib\charsets.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\deploy.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\javaws.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\jce.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\jfr.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\jfxswt.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\jsse.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\management-agent.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\plugin.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\resources.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\rt.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\access-bridge-64.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\cldrdata.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\dnsns.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\jaccess.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\jfxrt.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\localedata.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\nashorn.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\sunec.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\sunjce_provider.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\sunmscapi.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\sunpkcs11.jar;C:\Program Files\Java\jdk1.8.0_20\jre\lib\ext\zipfs.jar;D:\WorkSpace\GitPjs\RapidoidTest\target\classes;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-web\5.1.0\rapidoid-web-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-gui\5.1.0\rapidoid-gui-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-html\5.1.0\rapidoid-html-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-render\5.1.0\rapidoid-render-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-jpa\5.1.0\rapidoid-jpa-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-http-server\5.1.0\rapidoid-http-server-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-http-fast\5.1.0\rapidoid-http-fast-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-net\5.1.0\rapidoid-net-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-buffer\5.1.0\rapidoid-buffer-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-inject\5.1.0\rapidoid-inject-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-http-client\5.1.0\rapidoid-http-client-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-commons\5.1.0\rapidoid-commons-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-essentials\5.1.0\rapidoid-essentials-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\javassist\javassist\3.20.0-GA\javassist-3.20.0-GA.jar;C:\Users\sagnitude\.m2\repository\com\fasterxml\jackson\core\jackson-databind\2.6.6\jackson-databind-2.6.6.jar;C:\Users\sagnitude\.m2\repository\com\fasterxml\jackson\core\jackson-annotations\2.6.0\jackson-annotations-2.6.0.jar;C:\Users\sagnitude\.m2\repository\com\fasterxml\jackson\core\jackson-core\2.6.6\jackson-core-2.6.6.jar;C:\Users\sagnitude\.m2\repository\com\fasterxml\jackson\module\jackson-module-afterburner\2.6.6\jackson-module-afterburner-2.6.6.jar;C:\Users\sagnitude\.m2\repository\com\fasterxml\jackson\dataformat\jackson-dataformat-yaml\2.6.6\jackson-dataformat-yaml-2.6.6.jar;C:\Users\sagnitude\.m2\repository\org\yaml\snakeyaml\1.15\snakeyaml-1.15.jar;C:\Users\sagnitude\.m2\repository\javax\inject\javax.inject\1\javax.inject-1.jar;C:\Users\sagnitude\.m2\repository\org\apache\httpcomponents\httpasyncclient\4.1\httpasyncclient-4.1.jar;C:\Users\sagnitude\.m2\repository\org\apache\httpcomponents\httpcore\4.4.1\httpcore-4.4.1.jar;C:\Users\sagnitude\.m2\repository\org\apache\httpcomponents\httpcore-nio\4.4.1\httpcore-nio-4.4.1.jar;C:\Users\sagnitude\.m2\repository\org\apache\httpcomponents\httpclient\4.4.1\httpclient-4.4.1.jar;C:\Users\sagnitude\.m2\repository\commons-codec\commons-codec\1.9\commons-codec-1.9.jar;C:\Users\sagnitude\.m2\repository\commons-logging\commons-logging\1.2\commons-logging-1.2.jar;C:\Users\sagnitude\.m2\repository\org\apache\httpcomponents\httpmime\4.5\httpmime-4.5.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-watch\5.1.0\rapidoid-watch-5.1.0.jar;C:\Users\sagnitude\.m2\repository\org\rapidoid\rapidoid-integrate\5.1.0\rapidoid-integrate-5.1.0.jar;C:\Program Files (x86)\JetBrains\IntelliJ IDEA 2016.1.1\lib\idea_rt.jar" org.test.RapidoidRunner
Connected to the target VM, address: '127.0.0.1:25401', transport: 'socket'
INFO | main | org.rapidoid.config.RapidoidInitializer | Starting Rapidoid... | version = 5.1.0 | process = 37448@SAGA-PC | dir = D:\WorkSpace\GitPjs\RapidoidTest
INFO | main | org.rapidoid.setup.App | Inferring application root | main = org.test.RapidoidRunner | package = org.test
INFO | main | org.rapidoid.scan.ClasspathUtil | Scanning classpath | annotated = [@Controller, @Service, @Run, @Named, @Singleton] | packages = [org.test]
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files%20(x86)\JetBrains\IntelliJ%20IDEA%202016.1.1\lib\idea_rt.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\charsets.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\deploy.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\access-bridge-64.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\cldrdata.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\dnsns.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\jaccess.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\jfxrt.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\localedata.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\nashorn.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunec.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunjce_provider.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunmscapi.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunpkcs11.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\zipfs.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\javaws.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jce.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jfr.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jfxswt.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jsse.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\management-agent.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\plugin.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\resources.jar
WARN | main | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\rt.jar
INFO | main | org.rapidoid.scan.ClasspathUtil | Finished classpath scan | time = 182ms | searched = 1 | found = []
INFO | main | org.rapidoid.setup.WatchForChanges | Watching classpath for changes... | classpath = [D:\WorkSpace\GitPjs\RapidoidTest\target\classes]
INFO | watcher1 | org.rapidoid.setup.App | Detected class or resource changes
INFO | server | org.rapidoid.net.impl.RapidoidServerLoop | Server started | address = 0.0.0.0 | port = 5432 | I/O workers = 8 | accept = non-blocking
INFO | main | org.rapidoid.http.impl.HttpRoutesImpl | Registering handler | setup = app | verbs = GET | path = /showUri | roles = [] | tx = NONE | handler = json((Req) -> ...)
INFO | server1 | org.rapidoid.util.Msc | ----------------------------------
INFO | server1 | org.rapidoid.util.Msc | Restarting the web application...
INFO | server1 | org.rapidoid.util.Msc | ----------------------------------
INFO | server1 | org.rapidoid.ioc.IoCContextImpl | Resetting IoC context | context = []
INFO | server1 | org.rapidoid.ioc.IoCContextImpl | Resetting IoC context | context = []
INFO | server1 | org.rapidoid.scan.ClasspathUtil | Scanning classpath | annotated = [@Controller, @Service, @Run, @Named, @Singleton] | packages = [org.test]
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files%20(x86)\JetBrains\IntelliJ%20IDEA%202016.1.1\lib\idea_rt.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\charsets.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\deploy.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\access-bridge-64.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\cldrdata.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\dnsns.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\jaccess.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\jfxrt.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\localedata.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\nashorn.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunec.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunjce_provider.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunmscapi.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\sunpkcs11.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\ext\zipfs.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\javaws.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jce.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jfr.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jfxswt.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\jsse.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\management-agent.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\plugin.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\resources.jar
WARN | server1 | org.rapidoid.scan.ClasspathUtil | Classpath entry doesn't exist: C:\Program%20Files\Java\jdk1.8.0_20\jre\lib\rt.jar
INFO | server1 | org.rapidoid.scan.ClasspathUtil | Finished classpath scan | time = 55ms | searched = 1 | found = []
INFO | server1 | org.rapidoid.http.impl.HttpRoutesImpl | Registering handler | setup = app | verbs = GET | path = /showUri | roles = [] | tx = NONE | handler = json((Req) -> ...)
INFO | server1 | org.rapidoid.setup.App | Successfully restarted the application!
ERROR | server1 | org.rapidoid.net.impl.RapidoidWorker | Failed to process message! | message = null
java.lang.ExceptionInInitializerError
	at org.rapidoid.http.FastHttp.onRequest(FastHttp.java:86)
	at org.rapidoid.setup.AppRestartProcessor.onRequest(AppRestartProcessor.java:45)
	at org.rapidoid.http.FastHttpProtocol.process(FastHttpProtocol.java:72)
	at org.rapidoid.net.impl.RapidoidWorker.processNext(RapidoidWorker.java:192)
	at org.rapidoid.net.impl.RapidoidWorker.processMsgs(RapidoidWorker.java:153)
	at org.rapidoid.net.impl.RapidoidWorker.process(RapidoidWorker.java:145)
	at org.rapidoid.net.impl.RapidoidWorker.readOP(RapidoidWorker.java:137)
	at org.rapidoid.net.impl.AbstractEventLoop.processKey(AbstractEventLoop.java:89)
	at org.rapidoid.net.impl.AbstractEventLoop.insideLoop(AbstractEventLoop.java:138)
	at org.rapidoid.net.impl.AbstractLoop.run(AbstractLoop.java:65)
	at org.rapidoid.net.impl.RapidoidWorkerThread.run(RapidoidWorkerThread.java:65)
Caused by: java.lang.RuntimeException: 
	at org.rapidoid.u.U.rte(U.java:370)
	at org.rapidoid.u.U.rte(U.java:374)
	at org.rapidoid.commons.AutoExpandingMap.get(AutoExpandingMap.java:58)
	at org.rapidoid.render.Templates.fromFile(Templates.java:44)
	at org.rapidoid.http.impl.HttpIO.<clinit>(HttpIO.java:69)
	... 11 more
Caused by: java.lang.IllegalArgumentException: The file '404.html' doesn't exist! Path: [, templates, default\templates]
	at org.rapidoid.u.U.illegal(U.java:386)
	at org.rapidoid.u.U.must(U.java:419)
	at org.rapidoid.io.Res.mustExist(Res.java:261)
	at org.rapidoid.render.Templates$1.map(Templates.java:39)
	at org.rapidoid.render.Templates$1.map(Templates.java:36)
	at org.rapidoid.commons.AutoExpandingMap.get(AutoExpandingMap.java:56)
	... 13 more
java.lang.NoClassDefFoundError: Could not initialize class org.rapidoid.http.impl.HttpIO
	at org.rapidoid.http.FastHttp.onRequest(FastHttp.java:86)
	at org.rapidoid.setup.AppRestartProcessor.onRequest(AppRestartProcessor.java:45)
	at org.rapidoid.http.FastHttpProtocol.process(FastHttpProtocol.java:72)
	at org.rapidoid.net.impl.RapidoidWorker.processNext(RapidoidWorker.java:192)
	at org.rapidoid.net.impl.RapidoidWorker.processMsgs(RapidoidWorker.java:153)
	at org.rapidoid.net.impl.RapidoidWorker.process(RapidoidWorker.java:145)
	at org.rapidoid.net.impl.RapidoidWorker.readOP(RapidoidWorker.java:137)
	at org.rapidoid.net.impl.AbstractEventLoop.processKey(AbstractEventLoop.java:89)
	at org.rapidoid.net.impl.AbstractEventLoop.insideLoop(AbstractEventLoop.java:138)
	at org.rapidoid.net.impl.AbstractLoop.run(AbstractLoop.java:65)
	at org.rapidoid.net.impl.RapidoidWorkerThread.run(RapidoidWorkerThread.java:65)
ERROR | server2 | org.rapidoid.net.impl.RapidoidWorker | Failed to process message! | message = Could not initialize class org.rapidoid.http.impl.HttpIO
ERROR | server3 | org.rapidoid.net.impl.RapidoidWorker | Failed to process message! | message = Could not initialize class org.rapidoid.http.impl.HttpIO
java.lang.NoClassDefFoundError: Could not initialize class org.rapidoid.http.impl.HttpIO
	at org.rapidoid.http.FastHttp.onRequest(FastHttp.java:86)
	at org.rapidoid.setup.AppRestartProcessor.onRequest(AppRestartProcessor.java:45)
	at org.rapidoid.http.FastHttpProtocol.process(FastHttpProtocol.java:72)
	at org.rapidoid.net.impl.RapidoidWorker.processNext(RapidoidWorker.java:192)
	at org.rapidoid.net.impl.RapidoidWorker.processMsgs(RapidoidWorker.java:153)
	at org.rapidoid.net.impl.RapidoidWorker.process(RapidoidWorker.java:145)
	at org.rapidoid.net.impl.RapidoidWorker.readOP(RapidoidWorker.java:137)
	at org.rapidoid.net.impl.AbstractEventLoop.processKey(AbstractEventLoop.java:89)
	at org.rapidoid.net.impl.AbstractEventLoop.insideLoop(AbstractEventLoop.java:138)
	at org.rapidoid.net.impl.AbstractLoop.run(AbstractLoop.java:65)
	at org.rapidoid.net.impl.RapidoidWorkerThread.run(RapidoidWorkerThread.java:65)

```

**Comment by nmihajlovski** [comment]
Thanks for reporting, I fixed the bug (in v5.1.1). Please confirm the fix so I can close the issue.

**Comment by sagnitude** [comment]
Now the error is gone.
